tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = py27, pypy
+envlist = py27, pypy, py34
 
 [testenv]
 deps = 
     pytest
     pytest-capturelog
+    tzlocal
+    configobj
```

Comment:
You're right, I seem to have misunderstood something during my initial testing. I'll remove these.